test/switch_test.js

```diff
@@ -201,7 +249,7 @@ describe.skip('switch', () => {
     `);
   });
 
-  it('works with the switch from the CoffeeScript demo page', () => {
+  it.skip('works with the switch from the CoffeeScript demo page', () => {
```

Comment:
Usually this has to do with the editing not happening in the order you expect. I'll take a look today if I get a chance.